core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -117,8 +117,8 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.setObjective = async function(objective) {
-    await explorationObjectiveInput.clear();
-    await explorationObjectiveInput.sendKeys(objective);
+    await action.sendKeys(
```

Comment:
`sendKeys` doesn't clear the element, so I think we need an `action.clear` above here

Reply:
Done!